### PR TITLE
Couple of interface fixes

### PR DESCRIPTION
**Pull request by quardbreak**
## Description of changes
Part of https://github.com/NebulaSS13/Nebula/pull/2266
Used some changes for staging fixes.

## Why and what will this PR improve
Fixes tooltips for non-square view.
A bit nicer `/client/proc/toggle_fullscreen()` and `getviewsize()`.

## Authorship
Me.

### CODE REVIEW

File: code/modules/tooltip/tooltip.html [hunk 1]
```diff
@@ -93,7 +93,7 @@
 	<div id="wrap" class="wrap">
 		<div id="content" class="content"></div>
 	</div>
-	<script type="text/javascript" src="https://code.jquery.com/jquery-1.11.3.min.js"></script>
+	<script type="text/javascript" src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
```
Comment (quardbreak) [inline code/modules/tooltip/tooltip.html:96]: I think this should not be depended to external distributors. What about keeping this locally in codebase and compile with all assets?

Comment (MistakeNot4892) [reply code/modules/tooltip/tooltip.html:96]: I am against bundling other libraries with our codebase. I'm honestly not clear why we need jquery for tooltips at all.

File: code/modules/tooltip/tooltip.dm [hunk 2]
```diff
@@ -3,113 +3,121 @@ Tooltips v1.1 - 22/10/15
...
-	if(queueHide)
+	if (queueHide)
 		hide()
 
 	return TRUE
 
+
 /datum/tooltip/proc/hide()
-	if(queueHide)
+	queueHide = showing ? TRUE : FALSE
```
Comment (MistakeNot4892) [inline code/modules/tooltip/tooltip.dm]: `queueHide = !!showing`

File: code/modules/tooltip/tooltip.html (hunk 1 above)
Comment (afterthought2) [reply code/modules/tooltip/tooltip.html:96]: I think the target IE versions don't support sane js async handling, so while you aren't forced to use jquery per se it's not unreasonable to. We do bundle jquery with assets already (version shouldn't be super relevant), so could access it from there; the method here might be more reliable due to shitty dm asset distribution though.

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**